recipes/doubletd/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "doubletD" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/elkebir-group/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: eeba08ff5b3d0b7e6add7ddc0844a83454c36a3973cbb28d599705c16a37dcb1
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python >=3.6
+    - setuptools
+  host:
```

Comment:
Please remove the `build:` section and put `setuptools` in the `host:` section.

Reply:
Thanks,  the build section is now removed and `setuptools` is moved.